website/docs/guides/global-events.md

```diff
@@ -0,0 +1,111 @@
+---
+sidebar_position: 6
+---
+
+# Global Events
+
+In this section, we will delve into the concept of Global Events in KafkaFlow, which provides a mechanism to subscribe to various events that are triggered during the message production and consumption processes. 
+
+KafkaFlow offers a range of Global Events that you can subscribe to. These events can be used to monitor and react to different stages of message handling. Below, you'll find a list of available events:
```

Comment:
A suggestion:

KafkaFlow offers a range of Global Events that can be subscribed to. These events can be used to monitor and react to different stages of message handling. Below is a list of available events:

Reply:
Changed, please check.